apps/cloudappclient/manager.js

```diff
@@ -3,14 +3,15 @@
 var EventEmitter = require('events').EventEmitter
 var inherits = require('util').inherits
 var logger = require('logger')('cloudAppClient-manager')
-var eventRequest = require('./eventRequestApi')
 var eventRequestMap = require('./eventRequestMap.json')
+var eventRequestApi = require('./eventRequestApi')
```

Comment:
Could you please revert the change on this line? We should disallow that unneeded changes.